modules/rosa-updating-rosa-cli.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
+[id="rosa-updating-the-rosa-cli_{context}"]
+= Updating the ROSA CLI
+
+Update to the latest compatible version of the {product-title} CLI.
+
+.Procedure
+
+. Confirm that a new version of the ROSA CLI is available.
++
+[source,terminal]
+----
+$ rosa version
+----
++
+.Example
+[source,terminal]
+----
+$ rosa version
```

Comment:
Just like an earlier comment, the command and output should be separated. So since you already have a code block above that shows "$ rosa version", you can get rid of it in this one, and you can change the title to `.Example output`. Like:

```
.Example output
[source,terminal]
----
1.2.12
There is a newer release version '1.2.15', please consider updating: https://mirror.openshift.com/pub/openshift-v4/clients/rosa/latest/
----
```

Reply:
Done, thankyou!